app/build.gradle

```diff
@@ -201,5 +201,5 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
-    buildToolsVersion buildToolsVersion
+    buildToolsVersion '28.0.3'
 }
```

Comment:
This issue shouldn't etid build.gradle file, since this is irrelevant. Please revert this change.

Reply:
Fixed in the next commit